app/views/addons/index.html.erb

```diff
@@ -0,0 +1,68 @@
+<div class="content">
+  <section class="page-content container-fluid">
+
+    <%= link_to new_addon_path,class:"btn btn-info btn-floating col-md-2 btn-prime" do %>
+      <span>+ Add Addon</span>
+    <% end %></br>
+
+    <div class="row">
+      <div class="col-12">
+        <div class="card">
+          <h5 class="card-header">Data Tables</h5>
+
+          <div class="card-body">
+            <table id="bs4-table" class="table table-striped table-bordered tbl" style="width:100%">
+              <thead>
+                <tr>
+                  <th>Addon #</th>
+                  <th>Title</th>
+                  <th>Price</th>
+                  <th>Available</th>
+                  <th>Actions</th>
+                </tr>
+              </thead>
+
+              <tbody>
+                <% i = 1 %>
+
+                <% @addons.each do |addon|%>
+                  <tr>
+                    <td><%= i %></td>
+                    <td><%= addon.title.titleize %></td>
+                    <td><%= addon.price %></td>
+                    <td>
+                      <span class="<%= addon.is_available ? 'badge badge-primary': "badge badge-secondary" %>">
+                        <%= addon.is_available ? "Yes": "No" %>
+                      <span>
+                    </td>
+                    <td class= "flex-icons">
+                      <div class="icon" >
+                        <%= link_to  addon_path(addon),class: "tile-primary" do %>
+                          <i class="zmdi zmdi-open-in-new zmdi-hc-fw"></i>
+                        <% end %>
+                      </div>
+
+                      <div class="icon" >
+                        <%= link_to  edit_addon_path(addon),class: "tile-primary" do %>
+                          <i class="zmdi zmdi-edit zmdi-hc-fw"></i>
+                        <% end %>
+                      </div>
+
+                      <div class="icon" >
+                        <%= link_to  addon_path(addon),method: :delete,class: "tile-primary",data: {confirm: 'Are you sure?'} do %>

```

Comment:
Indentation!
`<%= link_to  addon_path(addon), method: :delete, class: "tile-primary", data: { confirm: 'Are you sure?' } do %>`

Reply:
{} always have space inside corners.

Reply:
done